src/test/scala/loamstream/compiler/LoamCompilerMultiFileTest.scala

```diff
@@ -37,87 +41,131 @@ final class LoamCompilerMultiFileTest extends FunSuite {
     assert(cmdTool.tokens.head.toString() === "echo Hello the answer is 42")
   }
 
-  private def createNewCompiler: LoamCompiler = LoamCompiler.default
-
-  private val scriptValues = LoamScript("values",
-    """
-      |val greeting = "Hello"
-      |val answer = 42
-    """.stripMargin)
-
+  private def valuesScript(scriptType: ScriptType): LoamScript = {
+    val name = "values"
+    val code = """|val greeting = "Hello"
+                  |val answer = 42""".stripMargin
+                  
+    scriptType match {
+      case ScriptType.Loam => LoamLoamScript(name, code)
+      case ScriptType.Scala => ScalaLoamScript(name, LoamCompilerTest.wrapInLoamFile(name, code))
+    }
+  }
+  
   test("Full name instead of import") {
-    val scriptIndividualImport = LoamScript("individualImport",
-      """
-        |cmd"echo ${values.greeting} the answer is ${values.answer}"
-      """.stripMargin)
-    val project = LoamProject(TestHelpers.config, scriptValues, scriptIndividualImport)
-    val compiler = createNewCompiler
-    val compileResults = compiler.compile(project)
-    assertCompiledFine(compileResults, 0, 1)
-    assertEchoCommand(compileResults)
+    def doTest(scriptType: ScriptType): Unit = {
```

Comment:
I don't know if it's easy to do (or maybe it's there and I don't see it), but it'd probably be slick if the examples at lease some of the examples coded up were used in the tests.

Reply:
What do you mean?  Do you mean running the jobs described in the example Loam files?  This test "just" checks that all the examples compile.

Reply:
I just picked a test at random. I just thought a test that actually ran one of the Loam examples that did several things to show a working script and the user see what the output should be, etc. would be useful.